docs/ONNX_Runtime_for_Mobile_Platforms.md

```diff
@@ -10,33 +10,65 @@ The minimal build can be used with any ORT format model, provided that the kerne
 
 You will need a script from the the ORT repository, and to perform a custom build, so you will need to clone the repository locally. See [here](https://github.com/microsoft/onnxruntime/blob/master/BUILD.md#prerequisites) for initial steps.
 
-Perform the following steps to create a minimal build from an ONNX format model.
+Perform the following steps to create a minimal build of ORT that is model specific. 
 
-### Create ORT format model
+### 1. Create ORT format model
 
-We will use a helper python script to convert an existing ONNX format model into an optimized ORT format model.
-This will require the ORT python package to be installed, and the ORT repository to have been cloned.
+We will use a helper python script to convert an existing ONNX format model into an ORT format model.
+This will require the ORT python package to be installed, and the ORT repository to have been cloned. 
```

Comment:
Globally change _ORT_ to _ONNX Runtime_ when referring to the runtime. Since we are introducing a new ORT model format which is the flat buffer format lets stick to consistent naming.